overlord/fdestate/fdemgr.go

```diff
@@ -88,3 +122,10 @@ func fdeMgr(st *state.State) *FDEManager {
 	}
 	return c.(*FDEManager)
 }
+
+func (m *FDEManager) doEFISecurebootDBUpdate(task *state.Task, tomb *tomb.Tomb) error {
+	// the handler does not do anything, it merely represents an action running externally
+
+	// TODO should this keep returning state.Retry?
+	return &state.Retry{}
```

Comment:
from discussion with @pedronis, let's not use Retry here, but rather keep using the blocking predicate to unblock the task, in which case the actual status of external operations could be cleanly picked up from the fdestate 'job' entry